template/scripts/build.sh

```diff
@@ -0,0 +1,31 @@
+
+# global variables
+
+echo "Enter project name:"
```

Comment:
Since it's a template, all these questions need to come from the `meta.js` file.
Ask the user these questions when he initiates the template. Put the results into the .env file, and then load it to the script using the same line of code from the twitter template.